src/components/ContainerSettingsVolumes.react.js

```diff
@@ -82,16 +91,16 @@ var ContainerSettingsVolumes = React.createClass({
     var homeDir = util.isWindows() ? util.windowsToLinuxPath(util.home()) : util.home();
     var mounts= _.map(this.props.container.Mounts, (m, i) => {
       let source = m.Source, destination = m.Destination;
-      if (!m.Source || m.Source.indexOf(homeDir) === -1) {
+      /*if (!m.Source || m.Source.indexOf(homeDir) === -1) {
```

Comment:
Don't comment out code, simply take it out - We have git versioning to roll back if needed